### PR TITLE
To do list with items

### PR DESCRIPTION
**Pull request by fjrf16**
Pull Request of the exercise of to do list separated with items

### CODE REVIEW

File: src/js/component/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+const Home = () => {
+	return (
+		<>
+			<List placeholder="List of To Do..."></List>
+		</>
+	);
+};
+
+export default Home;
```
Comment (gaal1le0) [inline src/js/component/Home.jsx]: I think you have not listened my requirements. Please try to explain me what is the main functionality of this component `Home`?
I know you don't know what are you doing, this software layer called `Home` gives to your code more complexity and it's not exactly what I want. 
You must change it and adapt your code to acomplish this issue.

File: src/js/index.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ import "bootstrap";
 import "../styles/index.scss";
 
 //import your own components
-import Home from "./component/home.jsx";
+import Home from "./component/Home.jsx";
 
 //render your react application
```
Comment (gaal1le0) [inline src/js/index.js]: Why have you commented this code?
Please remove all the comments.

File: src/styles/List.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
+/* Include the padding and border in an element's total width and height */
+* {
+    box-sizing: border-box;
+  }
+  /* Style the close button */
```
Comment (gaal1le0) [inline src/styles/List.scss]: Why have you commented this code?
I mean its obvious.

```css
.header {}
```

The above code styles header, I know, it's not a mandatory comment it, so it's a mandatory NOT COMMENT OBVIUS CODE. Please remove all unnecessary comments

File: src/styles/ListItem.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
+/* Remove margins and padding from the list */
```
Comment (gaal1le0) [inline src/styles/ListItem.scss]: Why have you commented this code?

File: src/styles/List.scss [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  .close {
+    position: absolute;
+    right: 0;
+    top: 0;
+    padding: 12px 16px 12px 16px;
```
Comment (gaal1le0) [inline src/styles/List.scss]: Why did you use this sentence?
```css
padding: 12px 16px;
```

It's more clear and better right?

**Review by gaal1le0 — CHANGES_REQUESTED** [review]
You have not accomplish my requirements to have this code perfect. Please review all my comments and change whatever you need. 
If you have any question about my comments please write here in a comment your question.

File: src/js/component/Home.jsx (hunk 1 above)
Comment (fjrf16) [reply src/js/component/Home.jsx]: I'm deleting the component Home, unnecesary component in the global functionality of the ToDoList

File: src/js/index.js (hunk 2 above)
Comment (fjrf16) [reply src/js/index.js]: I haved removed the unnecesary comments along the files.

File: src/styles/List.scss (hunk 3 above)
Comment (fjrf16) [reply src/styles/List.scss]: I haved remove all the unnecesary comments from the files.

File: src/styles/ListItem.scss (hunk 4 above)
Comment (fjrf16) [reply src/styles/ListItem.scss]: I have removed the comment.

File: src/styles/List.scss (hunk 5 above)
Comment (fjrf16) [reply src/styles/List.scss]: The sentence has been modified to the indicated.

**Comment by gaal1le0** [comment]
Then it's okey. Proceeded to merge

**Pull request merged**